stream/distributedlog/core/src/main/java/org/apache/distributedlog/impl/metadata/ZKLogStreamMetadataStore.java

```diff
@@ -680,6 +680,40 @@ public void processResult(int rc, String path, Object ctx) {
         return promise;
     }
 
+    @Override
+    public CompletableFuture<Boolean> deleteLogUnRecursive(URI uri, final String logName) {
```

Comment:
the distributed log will create a lot of metadata paths under the log, you can check the getLog method. Looks like the deleteLogUnRecursive will fail because we only delete the log path?

Reply:
> the distributed log will create a lot of metadata paths under the log, you can check the getLog method. Looks like the deleteLogUnRecursive will fail because we only delete the log path?

ok